SPECS/grub2/grub2.spec

```diff
@@ -60,6 +60,8 @@ Patch1009:      CVE-2020-14310.nopatch
 Patch1010:      CVE-2020-14311.nopatch
 Patch1011:      CVE-2020-27749.nopatch
 Patch1012:      CVE-2021-20225.nopatch
+# TPM measurement fix
+Patch1013:      remove-shim-lock-verification.patch
```

Comment:
Please move the patch to 0167. The goal here is to group functional patches in the <1000 patch number and CVE patches >1000